code/game/gamemodes/changeling/powers/linglink.dm

```diff
@@ -0,0 +1,73 @@
+/obj/effect/proc_holder/changeling/linglink
+	name = "Hivemind Link"
+	desc = "Link your victim's mind into the hivemind for personal interrogation"
+	chemical_cost = 0
+	dna_cost = 0
+	req_human = 1
+	max_genetic_damage = 100
+
+/obj/effect/proc_holder/changeling/linglink/can_sting(mob/living/carbon/user)
+	if(!..())
+		return
+	var/datum/changeling/changeling = user.mind.changeling
+	var/obj/item/weapon/grab/G = user.get_active_hand()
+
+	if(changeling.islinking)
+		to_chat(user, "<span class='warning'>We have already formed a link with the victim!</span>")
+		return
+	if(!istype(G))
+		to_chat(user, "<span class='warning'>We must be tightly grabbing a creature in our active hand to link with them!</span>")
+		return
+	if(G.state <= GRAB_AGGRESSIVE)
+		to_chat(user, "<span class='warning'>We must have a tighter grip to link with this creature!</span>")
+		return
+
+	var/mob/living/carbon/target = G.affecting
```

Comment:
This needs safety check; there's nothing to prevent you from doing this to a simple animal.